endpoints/HandoverEndpoint.ts

```diff
@@ -23,7 +22,7 @@ export class HandoverEndpoint extends ApiEndpoint {
 	): Promise<IApiResponse> {
 		console.log(InfoLogs.HANDOVER_ENDPOINT_REQUEST_RECEIVED);
 		try {
-			const assoc = new RocketChatAssociationRecord(RocketChatAssociationModel.MISC, `SFLAIA-${request.content.roomId}`);
+			const assoc = RoomAssoc(request.content.roomId);
```

Comment:
Since it's a function, it will be better if it starts with lower case to make it consistent. We can also change it to something like getRoomAssoc as the other getter functions follow similar convention

Reply:
Done ✅